.github/supporting-files/.gitignore

```diff
@@ -0,0 +1 @@
+!target/
```

Comment:
Can't remember why this is here, to be honest. Can you please explain it?

Reply:
It came from https://github.com/kiegroup/kogito-tooling/pull/401.